Doc/library/idle.rst

```diff
@@ -557,7 +557,7 @@ IDLE-console differences
 
 As much as possible, the result of executing Python code with IDLE is the
 same as executing the same code in a console window.  However, the different
-interface and operation occasionally affects visible results.  For instance,
+interface and operation occasionally affect visible results.  For instance,
```

Comment:
When I wrote this, I thought of 'different interface and operation' as one thing.  But if you did not see that, I agree with changing.